InBrowserComputerVision/ReactWebCamComputerVision/README.md

```diff
@@ -40,6 +43,9 @@ Hit F5 or run IIS Express. This should install required npm dependencies and lau
 Enter your credentials in the settings sidebar, click anywhere outside, and begin analyzing the webcam feed.
```

Comment:
Once you enter your credentials in the settings sidebar you will be ready to start processing images. To analyze a single image from the web cam feed, click on the "Analyze Single Frame" button. To analyze images from the web cam in a continuous loop, toggle the "Analyze Continuously" switch, as shown in the following animation.